src/Nethermind/Nethermind.BeaconNode/MockedStart/QuickStart.cs

```diff
@@ -163,7 +147,7 @@ public void QuickStartGenesis()
                 var deposit = new Deposit(proof, depositData);
 
                 _logger.LogDebug("Quick start adding deposit for mocked validator {ValidatorIndex} with public key {PublicKey}.",
-                    validatorIndex, publicKey);
+                    validatorIndex, publicKey.ToString().Substring(0, 12));
```

Comment:
I think there is a ToString / ToShortString so better add/use it consistently

Reply:
I will have a look at ToShortString(), which I presume is in your BlsPublicKey; but also the comment re logging.